tests/storage/reservation.go

```diff
@@ -360,6 +362,42 @@ var _ = Describe(SIG("SCSI persistent reservation", Serial, func() {
 			By("Ensuring all KubeVirt components are ready")
 			testsuite.EnsureKubevirtReady()
 		})
+
+		Context("With multipath", func() {
+			var mpathCleanUp bool
+			const mpathSocket = "/proc/1/root/run/multipathd.socket"
+			BeforeEach(func() {
+				// Check if mulitpathd socket exists on the nodes, if not simulate the existance by creating a mock socket
+				nodes := libnode.GetAllSchedulableNodes(virtClient)
+				for _, node := range nodes.Items {
+					_, err := libnode.ExecuteCommandInVirtHandlerPod(node.Name, []string{"ls", mpathSocket})
+					if err != nil {
+						By(fmt.Sprintf("Create a fake mulitpathd.socket in node %s", node.Name))
+						libnode.ExecuteCommandInVirtHandlerPod(node.Name, []string{"touch", mpathSocket})
+						mpathCleanUp = true
```

Comment:
More of a stylistic comment, but you could use `DeferCleanup` instead of keeping the state in `mpathCleanUp`.